files/zh-cn/web/api/document/createprocessinginstruction/index.md

```diff
@@ -1,44 +1,60 @@
 ---
-title: Document.createProcessingInstruction()
+title: Document：createProcessingInstruction() 方法
 slug: Web/API/Document/createProcessingInstruction
+l10n:
+  sourceCommit: acfe8c9f1f4145f77653a2bc64a9744b001358dc
 ---
 
 {{APIRef("DOM")}}
 
-`createProcessingInstruction()` 创建一个新的处理指令节点，并返回。
+`createProcessingInstruction()` 生成新的 [processing instruction](/zh-CN/docs/Web/API/ProcessingInstruction) 节点并返回。
```

Comment:
Should `processing instruction` conecept been translated?

Reply:
Yes